setup.go

```diff
@@ -347,6 +347,7 @@ func configFromEnv() Config {
 		switch opt {
 		case "stdout":
 			cfg.Stdout = true
+			cfg.Stderr = false
```

Comment:
thinking more, i think this is ngmi approach as if the user goes with `stderr+stdout` instead of `stdout+stderr`, then we will not set both 😓